mteb/abstasks/AbsTaskPairClassification.py

```diff
@@ -19,9 +21,20 @@ def evaluate(self, model, split="test", **kwargs):
         if not self.data_loaded:
             self.load_data()
 
-        data_split = self.dataset[split][0]
+        if self.is_multilingual:
+            scores = {}
+            for lang in self.dataset:
+                logger.info(f"\nTask: {self.description['name']}, split: {split}, language: {lang}. Running...")
+                scores[lang] = self._evaluate_monolingual(model, self.dataset[lang], split, **kwargs)
+        else:
+            logger.info(f"\nTask: {self.description['name']}, split: {split}. Running...")
+            scores = self._evaluate_monolingual(model, self.dataset, split, **kwargs)
+
+        return scores
 
-        logging.getLogger("sentence_transformers.evaluation.PairClassificationEvaluator").setLevel(logging.WARN)
+    def _evaluate_monolingual(self, model, dataset, split, **kwargs):
+        data_split = dataset[split][0]
```

Comment:
Pair Classification is supposed to be multilingual ? I have some trouble understanding the code.. 😢

Reply:
No, it's just the possibility to include datasets who are multilingual. Not cross lingual.

Reply:
And here i must change the way of loading this dataset with the "quality"
`            self.dataset[lang] = datasets.load_dataset(
                self.description["hf_hub_name"],
                lang=lang,
                quality=100,
                revision=self.description.get("revision", None),
            )`